src/projects/EnsureThat/Enforcers/BoolArg.cs

```diff
@@ -15,6 +15,17 @@ public bool IsTrue(bool value, [InvokerParameterName] string paramName = null, O
             return value;
         }
 
+        [ContractAnnotation("value:null=>true; value:false=>halt; value:true=>true")]
+        public bool? IsNullOrTrue(bool? value, [InvokerParameterName] string paramName = null, OptsFn optsFn = null)
```

Comment:
Please provide API compatibility for the other syntaxes.

Specifically, make sure these three can work [for each IsNullOrTrue, IsNullOrFalse, IsNullOrIn]:
```C#
bool? myInput = null;
Ensure.That(myInput ).IsNullOrTrue();
Ensure.Bool.IsNullOrTrue(myInput );
EnsureArg.IsNullOrTrue(myInput );
```